pkg/models/app.go

```diff
@@ -3,11 +3,11 @@ package models
 // App is the model struct for apps
 // swagger:model App
 type App struct {
-	ID                    string    `json:"id"`
-	AppID                 string    `json:"appId"`
-	AppName               string    `json:"appName"`
-	NumOfDeployedVersions int64     `json:"numOfDeployedVersions,omitempty"`
-	NumOfClients          int64     `json:"numOfClients,omitempty"`
-	NumOfAppLaunches      int64     `json:"numOfAppLaunches,omitempty"`
-	DeployedVersions      []Version `json:"deployedVersions,omitempty"`
+	ID                    string     `json:"id,omitempty"`
+	AppID                 string     `json:"appId,omitempty"`
+	AppName               string     `json:"appName,omitempty"`
+	NumOfDeployedVersions int        `json:"numOfDeployedVersions"`
+	NumOfClients          int        `json:"numOfClients"`
+	NumOfAppLaunches      int64      `json:"numOfAppLaunches"`
```

Comment:
Why just this one is the type int64 as the NumOfDeployedVersions and NumOfClients are int?

Reply:
Good spot, updated.